cli/types/cypress.d.ts

```diff
@@ -3550,6 +3550,7 @@ declare namespace Cypress {
     domain: string
     secure: boolean
     httpOnly: boolean
+    hostOnly: boolean
```

Comment:
Can we add descriptions for typescript to render for users?

Reply:
added in 58851a5ad7. Is this what you are referring to?

Reply:
I'd expect each property to be documented, so some of this is back-fill work 😬

Reply:
Wouldn't be we better off pointing them to the docs in the comment, which will have a more easily readable description for each property?

Reply:
I assume no because users want to see it inline when typing their options not clicking to the docs? I don't really leverage typescript annotations but I assume thats why people want them, everything upfront while coding.

Reply:
I think thats fair, especially since we do it for other types. I added them in b3c15859b5. Let me know what you think!